### DISCUSSION TITLE
[Feature] Multiple language support

### DISCUSSION THREAD

**Discussion by mprajescu** [mediacms-io/mediacms#608] [Feature Requests]
Another feature that would be great for this application is translation. 
I can help with Romanian translation since this is my native language.

**Comment by mgogoulos** [comment]
This is a big feature but something worth to consider!

**Comment by mgogoulos** [comment]
At some point we should make all strings as translatable on Django and React. This will involve translatable settings on python/html/javascript files but it is something Transifex can handle easily

**Comment by pxlwrk** [comment]
Is there a way to help you with that? I would be pleased to use a translated version.

**Comment by mgogoulos** [comment]
> Is there a way to help you with that? I would be pleased to use a translated version.

We're working on opening the frontend (React) code, once this is made we can mark all translatable content, upload to Transifex and request help for adding translations. 

Thanks

**Comment by malakhovks** [comment]
> > Is there a way to help you with that? I would be pleased to use a translated version.
> 
> We're working on opening the frontend (React) code, once this is made we can mark all translatable content, upload to Transifex and request help for adding translations.
> 
> Thanks

It will be great! I will translate it in Ukrainian.

**Comment by phreaknerd** [comment]
I'd take the german, swedish part.

**Comment by Dan1ell** [comment]
Dutch. Really waiting for this.

**Comment by pxlwrk** [comment]
> I'd take the german, swedish part.

I will support you with German translation.

**Comment by Avskum** [comment]
Czech ready here. Is there atleast now possibility to manualy translate stuff somehow? @mgogoulos ? :-)

**Reply by mgogoulos** [reply]
Unfortunately not, but promise this will be fixed soon. It is priority number 1

**Comment by cdorsat** [comment]
I think the function of supporting multiple languages should be added. The newly added language defaults to English, and the website administrator who uses it will translate it by himself. Translation files can be contributed to the community. Additionally, I am happy to provide Traditional Chinese and Simplified Chinese translations.

**Reply by Avskum** [reply]
I completly agree...I am dying for this feature .. I tryed multiple ways to do this, but I am not that good in Python sadly.

**Comment by mgogoulos** [comment]
They are now included in the latest release! Check it out on https://demo.mediacms.io/ and update!!!

Also if you want to help improving them, checkout https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#20-translations
